docs/tapiro/rpc.md

```diff
@@ -0,0 +1,6 @@
+---
+id: rpc
```

Comment:
id: why

for consistency?

Reply:
Link a blogpost è lì come tampone. Idea in un secondo momento era spiegare RPC, approccio etc, per questo l'ho chiamato così.

Reply:
👍